### DISCUSSION TITLE
unable to merge sync PRs

### DISCUSSION THREAD

**Discussion by orama254** [reactjs/reactjs.org-translation#255] [Team Posts]
Hi team...

I'd like assistance from anyone @reactjs/reactjs-org-localization on how to merge sync prs for https://github.com/reactjs/sw.reactjs.org

Your help will be appreciated

**Comment by smikitky** [comment]
If you are new to merging sync PRs, these are the basic steps:

1. Checkout the latest sync PR (e.g., `sync-84ad3308`).
2. Open each conflicted file (in your latest PR, `content/docs/hello-world.md`) and find the conflict markers.
3. See the upstream diff and resolve the conflicts accordingly. The easiest way to see the upstream diff is going to the "Files Changed" tab and changing the commits filter.
4. Commit to the sync branch (e.g., `git commit . -m "Resolve conflicts"`).
5. Push the branch, wait for the CI to run, and merge the PR.

**Comment by orama254** [comment]
Thanks for this @smikitky 

The other issue in relation to this is that I currently don't have write permission as displayed below
![write_permission_reactjsorg](https://user-images.githubusercontent.com/50571688/163704540-a326e917-d1b8-4214-b03d-f6d6417602d1.png)

Also the Swahili team is missing from the list of localization teams as demonstrated below
![swahili_team_missing](https://user-images.githubusercontent.com/50571688/163704570-4ad2bd5e-188d-4cca-b0de-53cd31aa6c0c.png)
 
 Any suggestions on how to go about it?

**Comment by smikitky** [comment]
@orama254 Maintainers of each fork should have full (admin) access to the fork. If you are one of the maintainers, contact the other maintainers. If you are the only maintainer of the Swahili language fork, you need to contact the React team.

**Comment by orama254** [comment]
Thanks for this again @smikitky,

I'll have to follow up with the React Team on this

**Comment by orama254** [comment]
Hi Team, still following up on this. any assistance will be appreciated.



cc @gaearon @tesseralis @react-translations-bot

**Comment by gaearon** [comment]
hi @orama254, is the issue that you're in the maintainers list but don't have permissions?

**Comment by orama254** [comment]
Yes @gaearon , I currently don't have write permission.

Plus also our translation team (reactjs.org Swahili translation) is not available currently.

**Comment by gaearon** [comment]
Hey @orama254, sorry for the frustration.

We're rebooting the translation effort now that the new site is done. So I'm closing all old issues, and will be watching for new issues if they pop up. Please file a new issue if the problem still exists — I'll be looking closely.

For now I've archived all stale translations. I realize that the permission issue is a part of why it didn't work. If you want to start the Swahili translation for the new site, I'll make sure that you have the permissions this time.

**Reply by orama254** [reply]
Hello @gaearon, Thanks for the clarification

I am super open to starting the translation process for the Swahili translation for the new documentation.

Let me know any additional steps needed from my side to make this happen. Looking forward.

**Reply by gaearon** [reply]
Please create a new PR in this repo to start the language, and we'll try to go through the process. Thanks!
